### ISSUE TITLE
Github remote team repo has server error 500

### ISSUE THREAD

**Issue by bokuanT** [nus-cs2103-AY2223S2/forum#298]
When trying to open my team's GitHub repo page, I get this error page:

![image](https://user-images.githubusercontent.com/97268566/228999768-c5adf1b1-0209-4453-912c-edbdf674209c.png)

I am using windows and chrome. I have tried clearing my browser cache and cookies, using another browser (opera), and accessing the page through my phone but I still get this error page. My teammates can access the page fine.

Another note was that I could access my team's Github page while I was logged out, but the same error appeared after logging into GitHub.

Has anyone else experienced this before or knows how to fix this?

**Comment by seanfirefox** [comment]
I have encountered the same problem too. Might be Github server issues.
![image](https://user-images.githubusercontent.com/35872524/229001264-811fc32d-7893-4322-b178-a161d6f0d539.png)

**Comment by valerietanhx** [comment]
Hi! A few of our team members are also experiencing this, though we're not sure how to fix it :")

**Comment by ScorpiusSigma** [comment]
This is a server error from github. You can get an updates on the situation here

https://www.githubstatus.com/incidents/rmc0cpn9sn8w

**Comment by damithc** [comment]
Yes, GitHub seems to be having issues. For the time being, we'll be extending the v1.3 jar submission deadline till 12 noon.

**Comment by damithc** [comment]
GitHub issue seems to be resolved now, as per https://www.githubstatus.com/
![image](https://user-images.githubusercontent.com/1673303/229007244-31c650dc-e2ba-43b3-b709-ac66076a888b.png)
